scripts/core/device.yml

```diff
@@ -144,6 +144,8 @@ params:
             [out][optional] pointer to the number of devices.
             pNumDevices will be updated with the total number of devices available.
 returns:
+    - $X_RESULT_ERROR_INVALID_SIZE:
+        - "`NumEntries == 0 && phDevices != NULL`"
```

Comment:
The description for `NumEntries` should be updated too since it says that it will return `UR_RESULT_ERROR_INVALID_VALUE` in this case

Reply:
Actually, I went with SIZE because that's what the test was using. But on second thought, maybe INVALID_VALUE is better?

Reply:
It would align with the OpenCL error code. What do other entry points specify for this type of error case?

Reply:
Consistency is best, yup.

Reply:
Having the same fault on HIP so 👍 from me

Reply:
Done :+1: